rsocket-core/src/main/java/io/rsocket/plugins/Plugins.java

```diff
@@ -0,0 +1,24 @@
+package io.rsocket.plugins;
+
+/** JVM wide plugins for RSocket */
```

Comment:
It's part of this PR, you can add interceptors to the RSocketFactory.  But I got the impression that it may be a useful feature to apply some plugins at app startup for all clients.  I'm not against that being used judiciously.